pytest_watch/watcher.py

```diff
@@ -77,6 +78,14 @@ def on_any_event(self, event):
         self.event_queue.put((type(event), src_path, dest_path))
 
 
+def _get_pytest_runner(custom):
+    if custom:
+        return custom.split(' ')
+    if os.getenv('VIRTUAL_ENV'):
+        return ('py.test',)
+    return (sys.executable, '-m', 'pytest')
```

Comment:
Just tested this locally. Got `TypeError: can only concatenate tuple (not "list") to tuple` because lists were used outside this function.

I just patched this in f4814f90883ba93a845cf58da08164d84959f718.